core/src/components/tab-button/tab-button.tsx

```diff
@@ -82,7 +83,7 @@ export class TabButton implements ComponentInterface, AnchorInterface {
     const dispatchedFrom = ev.target as HTMLElement;
     const parent = this.el.parentElement as EventTarget;
 
-    if ((ev.composedPath && ev.composedPath().includes(parent)) || (dispatchedFrom && dispatchedFrom.contains(this.el))) {
+    if (ev.composedPath?.().includes(parent) || dispatchedFrom?.contains(this.el)) {
```

Comment:
```suggestion
    if (ev.composedPath().includes(parent) || dispatchedFrom?.contains(this.el)) {
```
I don't think the `composedPath` check is necessary: https://caniuse.com/mdn-api_event_composedpath